src/routes/User.js

```diff
@@ -0,0 +1,12 @@
+import { UserController } from '../controllers';
+import Authorization from '../middlewares';
+import Validate from '../validation/User';
+
+const userRoutes = app => {
+  app.patch('/api/v1/updateuser',
+    Authorization.checkToken,
```

Comment:
Cool!, so you are using the middlewear here. You don't need to call verifyToken function in the controller you implemented. just get the decoded payload of the token from `req.decoded`.  Maybe you should take a second look at that middleware